pcsx2/USB/usb-python2/usb-python2.cpp

```diff
@@ -368,6 +389,39 @@ namespace usb_python2
 				mm_setKB(true);
 				#endif
 
+				#ifdef INCLUDE_BTOOLS
+				hDDRIO = LoadLibraryA("ddrio.dll");
+
+				if (hDDRIO != nullptr)
+				{
+					m_ddr_io_set_loggers = (ddr_io_set_loggers_type*)GetProcAddress(hDDRIO, "ddr_io_set_loggers");
+					m_ddr_io_init = (ddr_io_init_type*)GetProcAddress(hDDRIO, "ddr_io_init");
+				
+					m_ddr_io_set_lights_p3io = (ddr_io_set_lights_p3io_type*)GetProcAddress(hDDRIO, "ddr_io_set_lights_p3io");
+					
+					m_ddr_io_fini = (ddr_io_fini_type*)GetProcAddress(hDDRIO, "ddr_io_fini");
+
+					s->isUsingBtoolLights = m_ddr_io_set_loggers && m_ddr_io_init && m_ddr_io_fini;		
```

Comment:
Add `&& m_ddr_io_set_lights_p3io` to this chain.
```suggestion
					s->isUsingBtoolLights = m_ddr_io_set_loggers && m_ddr_io_init && m_ddr_io_fini && m_ddr_io_set_lights_p3io;		
```